### PR TITLE
Redesign the Modals to have consistent UI

### LINKED ISSUE #1020: Make sure all modals have consistent UI in the system
redirectLicenseModals

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/32403

**Changes**
- Redesign the licenses modal and redirect modal as per the designs outlined in the issue #1020 

image

closes #1020

### CODE REVIEW

**Comment by annielovesjs** [comment]
The redirect modal looks good but the licenses modal didnt seem to change, or did you mean to change only the redirect modal?
inconsistent

**Review by annielovesjs — APPROVED** [review]

**Comment by streetcoder123** [comment]
> The redirect modal looks good but the licenses modal didnt seem to change, or did you mean to change only the redirect modal?
> inconsistent

It is also changed. Should look like this
image

File: src/client/src/containers/ViewLicensesModal/index.tsx [hunk 1]
```diff
@@ -28,23 +30,38 @@ const messages = defineMessages({
...
+    id: "licenses.licenses",
+    defaultMessage: "Licenses"
   }
 });
 
 const ViewLicensesModal = ({ intl, closeModal }: Props) => {
   const { formatMessage } = intl;
 
+  const cancelKeyDownHandler = (event: React.KeyboardEvent<SVGSVGElement>) => {
+    if (event.key === KEY_EVENTS.ENTER || event.key === KEY_EVENTS.SPACE) {
```
Comment (Tanya0609) [inline src/client/src/containers/ViewLicensesModal/index.tsx:44]: Just curious, I was pressing enter/space, but nothing happen. How could I test this code?

**Review by Tanya0609 — APPROVED** [review]
modals are consistent! great work!

File: src/client/src/containers/ViewLicensesModal/index.tsx (hunk 1 above)
Comment (SahilTara) [reply src/client/src/containers/ViewLicensesModal/index.tsx:44]: 1) open the licenses modal
2) Click on it (the license modal)
3) Press tab until you highlight the "X" on the top right (the close svg)
4) Press enter/space
5) Observe the modal closing.

Hope this helps!

**Review by SahilTara — APPROVED** [review]

File: src/client/src/containers/ViewLicensesModal/index.tsx (hunk 1 above)
Comment (Tanya0609) [reply src/client/src/containers/ViewLicensesModal/index.tsx:44]: Thanks!

**Pull request merged**